js/skin.js

```diff
@@ -207,6 +207,17 @@ var Skin = React.createClass({
     return vrViewingDirectionValue
   },
 
+
+  /**
+   *
+   * @param comparableClass {string} - className for comparing
+   * @param originClass {string} - className for comparing
+   * @returns {boolean} returns true if classes are identical
+   */
+  areClassNamesEquals: function(comparableClass, originClass) {
```

Comment:
I think this is too generic to be in `skin.js`, this would probably be better inside `Utils`

Reply:
\+